FacialRecog.py

```diff
@@ -184,11 +181,16 @@ def all_cameras():
 """
 
 
-def get_frame_from_camera(camera):
-    feed = cv2.VideoCapture("http://" + str(camera['ip']) + "/video.mjpg")
-    ret, img = feed.read()
+def get_frame_from_camera(image_hub, camera):
+
+    camera_name, img = image_hub.recv_image()
+    image_hub.send_reply(b'OK')
+    while str(camera_name) not in str(camera['ip']):
+        camera_name, img = image_hub.recv_image()
+        image_hub.send_reply(b'OK')
+
     frame = cv2.resize(img, (0, 0), fx=0.75, fy=0.75)
-    return frame
+    return camera_name, frame
```

Comment:
do we need to return camera_name? because in RecogScript we get camera_ip, frame but never actually use camera_ip (it's already available in camera['ip'])

Reply:
True, we don't actually need to do that if we are checking to make sure the returned frame is from the right camera. Will get rid of and re-commit.